### ISSUE TITLE
Bug with Multiple Dynamic Responsive Sliders

### ISSUE THREAD

**Issue by hauran** [iosscripts/iosslider#250]
Thanks for this amazing product!

I think I found  a bug. If there is more than 1 responsive slider which adds slides dynamically, only the slider that has most recently had an addSlide method executed is responsive.  The other slider does not resize. 

When you begin, both are responsive. But if you click an "Add Slide" button, only that slider is responsive.

 I modified the provided responsive example to demonstrate the bug.

Let me know if you need any more info.
Thanks again.

script:
<code>

```
        $(document).ready(function() {
            $('.test1 .iosSlider').iosSlider({
                snapToChildren: true,
                desktopClickDrag: true,
                keyboardControls: true
            });

            $('.test2 .iosSlider').iosSlider({
                snapToChildren: true,
                desktopClickDrag: true,
                keyboardControls: true
            });

            slides = ["../../_resources/h-slider-2.jpg", "../../_resources/h-slider-3.jpg", "../../_resources/h-slider-4.jpg", "../../_resources/h-slider-5.jpg"];

            topCount = 0, bottomCount = 0;

            $('button.top').click(function(){
                numberOfSlides = $('.test1 .iosSlider').data().iosslider.numberOfSlides
                $('.test1 .iosSlider').iosSlider('addSlide', "<div class = 'item'><img src='" + slides[topCount] + "'/></div>" ,numberOfSlides+1)
                topCount++;
            })

            $('button.bottom').click(function(){
                numberOfSlides = $('.test2 .iosSlider').data().iosslider.numberOfSlides
                $('.test2 .iosSlider').iosSlider('addSlide', "<div class = 'item'><img src='" + slides[bottomCount] + "'/></div>" ,numberOfSlides+1)
                bottomCount++;
            })
        }); 
```

</code>

HTML:
<code>

```
    <button class='top'>Add Slide - Top</button>

    <button class='bottom'>Add Slide - Bottom</button>

    <div class='test1'>
        <div class = 'container'>
            <div class = 'iosSlider'>
                <div class = 'slider'>
                    <div class = 'item'>
                        <img src='../../_resources/h-slider-1.jpg'/>
                    </div>
                </div>
            </div>
        </div>
    </div>

    <div class='test2'>
        <div class = 'container'>
            <div class = 'iosSlider'>
                <div class = 'slider'>
                    <div class = 'item'>
                        <img src='../../_resources/h-slider-1.jpg'/>
                    </div>
                </div>
            </div>
        </div>
    </div>
```

</code>

**Comment by marcwhitbread** [comment]
Your CSS could be a factor. Can you provide CSS for this example as well?

**Comment by hauran** [comment]
I did not change the provided CSS in the 9-Full-Width-Responsive-Height example

<code>

```
    <link rel = "stylesheet" href = "_css/default.css" />
    <link rel = "stylesheet" media = "screen" href = "_css/common.css" />
```

</code>

**Comment by marcwhitbread** [comment]
Ah, I see it. Interesting issue. I'll take a look.

**Comment by marcwhitbread** [comment]
I fixed it in v1.2.12. Let me know if that works for you. Thanks again for the submission.
